openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/client/PortForwarder.java

```diff
@@ -245,10 +255,19 @@ public void close() {
     }
 
     public static void main(String[] args) throws Exception {
+
         if (args.length < 4) {
-            System.out.println("Usage: portforward <namespace> <pod> <source-port> <target-port>");
-            System.out.println("Example: portforward mynamespace somepod 8080 8080");
+            System.out.println("Usage: portforward <namespace> <pod> <source-port> <target-port> -b [optional] <bindAddress>");
+            System.out.println("Example: portforward mynamespace somepod 8080 8080 -b 10.1.1.1");
+
         }
+
+        for (int i = 0; i < args.length; i++) {
+            if (args[i].contains("-b")){
+                portForwardBindAddress = InetAddress.getByName(args[i+1]);
```

Comment:
Here you should call the set() method instead of assigning the variable directly.